charts/sysdig-deploy/CHANGELOG.md

```diff
@@ -51,7 +51,7 @@ This file documents all notable changes to Sysdig's sysdig-deploy Helm Chart. Th
 
 ### Minor changes
 * Bumped nodeAnalyzer to 1.7.5
-* Bumped benchmark runner to 1.0.18.2
+* Bumped benchmark runner to 1.0.19.0
```

Comment:
Please don't modify old changelog entries, you should add a new entry to the top of this file using the new `sysdig-deploy` version

Reply:
Same applies to the `node-analyzer` chart

Reply:
Done. added new entries with bumped version of `node-analyzer` and `sysdig-deploy`